ApiTests/canvasTLAdminTest.sh

```diff
@@ -0,0 +1,63 @@
+set +x
+
+function help {
+echo "This script tests the Canvas TL API which allows canvas requests using masquerading."
+echo "Copy the AUTH.sh.template file to AUTH.sh and fill in the appropriate tokens."
+echo "The CanvasTLAdmin ESB API will allow masquarding by default.  The direct Canvas account"
+echo "must be for a user with masquerade permissions."
+echo ""
+echo "If the right credentials are added the same queries can be run through different"
+echo "access channels. E.g. they can be run via direct canvas access or"
+echo "via access through a ESB API. See script for an example."
+echo "The results of queries through different channels should be the same except:"
+echo "- A query for the user actually running the query may give different information"
+echo "as ESB API supplies it's own account for making queries."
+echo "- The accounts for the direct and ESB quries need to have the same permissions in Canvas."
+}
+
+### Set the token and prefix values for the different users.
+source ./AUTH.sh
+
+help
+
+echo "++++++++++ testing canvas access $CHANNEL_NAME ++++++++++"
+
+function run_tests {
+    echo "############################# $CHANNEL_NAME ###################"
+#    echo "+++ trying self $CHANNEL_NAME +++"
```

Comment:
No. These are examples of different queries that might be useful, so they should stay as reference.  The style could be clearer but I'd like to see how much this code is used before more time is put into it.